### PR TITLE
Test change in order of error messages

### PR DESCRIPTION
**Pull request by car-roll**
Testing the change in error message printout from https://github.com/jenkinsci/workflow-job-plugin/pull/184

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by bitwiseman** [comment]
How do we interpret this result?

**Comment by car-roll** [comment]
Turned off FailFast

@bitwiseman I don't know if any of those failing tests in the the previous run were relevant? I guess we'll see once all the tests have completed if there is any test that is now failing due to the change in messages.

**Comment by timja** [comment]
```

[2021-03-22T16:50:14.560Z] Error : The git-checkout command failed. || Cloning into '/home/jenkins/workspace/Tools_bom_PR-468/pct-work/workflow-job'...

[2021-03-22T16:50:14.560Z] From git://github.com/jenkinsci/workflow-job-plugin

[2021-03-22T16:50:14.560Z]  * branch            HEAD       -> FETCH_HEAD

[2021-03-22T16:50:14.560Z] fatal: reference is not a tree: f84ef8f931377f0cad75a718b78f804159109070

[2021-03-22T16:50:14.560Z] 

[2021-03-22T16:50:14.560Z] Mar 22, 2021 4:50:14 PM org.jenkins.tools.test.PluginCompatTester testPlugins

[2021-03-22T16:50:14.560Z] SEVERE: Internal error while executing a test for core 2.277.1 and plugin workflow-job 2.41-rc1105.f84ef8f93137. Please submit a bug to plugin-compat-tester

[2021-03-22T16:50:14.560Z] org.jenkins.tools.test.exception.PluginSourcesUnavailableException: Problem while checking out plugin sources!

[2021-03-22T16:50:14.560Z] 	at org.jenkins.tools.test.PluginCompatTester.testPluginAgainst(PluginCompatTester.java:493)

[2021-03-22T16:50:14.560Z] 	at org.jenkins.tools.test.PluginCompatTester.testPlugins(PluginCompatTester.java:312)

[2021-03-22T16:50:14.560Z] 	at org.jenkins.tools.test.PluginCompatTesterCli.main(PluginCompatTesterCli.java:175)

[2021-03-22T16:50:14.561Z] Caused by: java.lang.RuntimeException: The git-checkout command failed. || Cloning into '/home/jenkins/workspace/Tools_bom_PR-468/pct-work/workflow-job'...

[2021-03-22T16:50:14.561Z] From git://github.com/jenkinsci/workflow-job-plugin

[2021-03-22T16:50:14.561Z]  * branch            HEAD       -> FETCH_HEAD

[2021-03-22T16:50:14.561Z] fatal: reference is not a tree: f84ef8f931377f0cad75a718b78f804159109070

[2021-03-22T16:50:14.561Z] 

[2021-03-22T16:50:14.561Z] 	at org.jenkins.tools.test.PluginCompatTester.cloneFromSCM(PluginCompatTester.java:606)

[2021-03-22T16:50:14.561Z] 	at org.jenkins.tools.test.PluginCompatTester.testPluginAgainst(PluginCompatTester.java:479)

[2021-03-22T16:50:14.561Z] 	... 2 more
```

File: bom-latest/pom.xml [hunk 1]
```diff
@@ -17,7 +17,7 @@
         <structs-plugin.version>1.22</structs-plugin.version>
         <workflow-api-plugin.version>2.42</workflow-api-plugin.version>
         <workflow-cps-plugin.version>2.90</workflow-cps-plugin.version>
-        <workflow-job-plugin.version>2.40</workflow-job-plugin.version>
+        <workflow-job-plugin.version>2.41-rc1105.f84ef8f93137</workflow-job-plugin.version>
```
Comment (bitwiseman) [inline bom-latest/pom.xml:20]: @car-roll 
```suggestion
        <workflow-job-plugin.version>2.41-rc1112.91a865190a8</workflow-job-plugin.version>
```

With incrementals fixed.

File: bom-latest/pom.xml [hunk 2]
```diff
@@ -17,7 +17,7 @@
         <structs-plugin.version>1.22</structs-plugin.version>
         <workflow-api-plugin.version>2.42</workflow-api-plugin.version>
         <workflow-cps-plugin.version>2.90</workflow-cps-plugin.version>
-        <workflow-job-plugin.version>2.40</workflow-job-plugin.version>
+        <workflow-job-plugin.version>2.41-rc1112.91a865190a8</workflow-job-plugin.version>
```
Comment (timja) [inline bom-latest/pom.xml:20]: 🤦 

```suggestion
        <workflow-job-plugin.version>2.41-rc1112.91a865190a8b</workflow-job-plugin.version>
```

**Comment by timja** [comment]
Tests pass

**Pull request closed without merging**